include/seqan/arg_parse/arg_parse_type_support.h

```diff
@@ -71,6 +93,23 @@ inline TTarget _cast(TString const s)
 }
 
 // ----------------------------------------------------------------------------
+// Function _castBool()
+// ----------------------------------------------------------------------------
+
+template <typename TString>
+inline bool _castBool(bool & dst, TString const s)
+{
+    // since the validValue check already verifies that the value if one of
+    // BooleanArgumentValues_<>::LIST, one only needs to check one equivalent class
+    // dst = std::find()
+    dst = (std::find(BooleanArgumentValues_<>::LIST_TRUE.begin(),
+                     BooleanArgumentValues_<>::LIST_TRUE.end(),
+                     s)
+           != BooleanArgumentValues_<>::LIST_TRUE.end());
+    return true;
```

Comment:
I would prefer, if we check both the negative and the positive value and return false if neither match. It forces people to use the interface right and makes detecting errors easier.